preload/prefetch-headers.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<title>Ensures that prefetch sends headers as per-spec</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="resources/prefetch-helper.js"></script>
+<body>
+<script>
+
+promise_test(async t => {
+    const {href} = await prefetch({"type": "image/png", file: "../../images/green.png"}, t);
+    const [info] = await get_prefetch_info(href);
+    const {headers} = info;
+    assert_equals(headers["sec-fetch-dest"], "empty");
```

Comment:
"empty" doesn't seem to be a valid destination per https://fetch.spec.whatwg.org/#concept-request-destination ?

Reply:
It's per spec https://w3c.github.io/webappsec-fetch-metadata/#abstract-opdef-set-dest